src/services/resource-service.ts

```diff
@@ -117,15 +117,28 @@ export const ResourceService = {
       })
     })
   },
-  updateAttachment: async (params?: UpdateAttachmentParams) =>
-    await axiosClient.patch(
+  updateAttachment: async (params?: UpdateAttachmentParams) => {
+    return await axiosClient.patch(
       createUrlPath(URLs.resources.attachments.patch, params?.id),
       params
-    ),
-  deleteAttachment: async (id: string): Promise<AxiosResponse> =>
-    await axiosClient.delete(
+    )
+  },
+  updateAttachmentQuery: (data: UpdateAttachmentParams) => {
+    const { id, ...attachmentData } = data
```

Comment:
Please, add a blank line between these statements
```suggestion
    const { id, ...attachmentData } = data

```